fuzzers/others/libafl-fuzz/Makefile.toml

```diff
@@ -66,15 +71,27 @@ test -d "./test/output/fuzzer_main/crashes" || {
 	echo "No crashes directory found"
 	exit 1
 }
+'''
+dependencies = ["build_afl"]
+
+[tasks.test_cmplog]
+script_runner = "@shell"
+script = '''
+cargo build --profile ${PROFILE}
 # cmplog TODO: AFL_BENCH_UNTIL_CRASH=1 instead of timeout 15s
 AFL_LLVM_CMPLOG=1 AFL_PATH=${AFL_DIR_NAME} ${AFL_CC_PATH} ./test/test-cmplog.c -o ./test/out-cmplog
```

Comment:
FWIW the way these makefiles _should_ work is that every compile thingy is one task, and then the task only gets run once.

So building cmplog is one task, while running the test should be another

Reply:
(... that depends on the build_cmplog task of course)

Reply:
The reason I have cargo build in each test is so that I can test individual elements. But I can move the cargo build in it's own task

Reply:
yes building and testing individual elements would be cleaner I think (for ppl that actually want to use this as a makefile..)